std/experimental/ndslice/slice.d

```diff
@@ -579,14 +592,31 @@ auto slice(
 ///
 pure nothrow unittest
 {
-    auto slice = slice!int(5, 6, 7);
-    assert(slice.length == 5);
-    assert(slice.elementsCount == 5 * 6 * 7);
-    static assert(is(typeof(slice) == Slice!(3, int*)));
+    auto sl = slice!int(5, 6, 7);
```

Comment:
Yep me too, but it's also pretty confusing when one reads the code (is the function or object) now called. how about `v` (for vector)?